### ISSUE TITLE
Add functionality that allows users to send links to specific dates and/or events 

### ISSUE THREAD

**Issue by TheJuanAndOnly99** [finos/calendar#127]
The ability to create links that point to specific events and/or dates would be very useful as it can be hard to parse through the calendar to find an event.
```[tasklist]
### Tasks
- [ ] month view
- [ ] week view
- [ ] day view
- [ ] selected event
- [ ] downloadable calendar event .ics
```

**Comment by gautamjuyal** [comment]
I think we'll need to introduce react router to the project, that way we can use event UID in the shareable URL, and parse the URL to access the event. Is there any other way to access route params/query apart from using react router?

**Comment by winmorgan** [comment]
Hi everyone. Just wanted to put a little background in here. 

1. The specific event link would be very useful for CTAs in general everywhere
2. Less important, but also useful would be the ability to call out a specific date as we could incorporate that into our weekly TWAF email reducing the number of links significantly. (Thursday Meetings --> click --> I'm on calendar for Thursday)

**Comment by mimiflynn** [comment]
This might be the same as issue #106

**Comment by maoo** [comment]
Thanks @mimiflynn and @gautamjuyal for the input, are you planning to work on a poc?

Agreed that this is a dup of #106, I'd suggest to keep the conversation here, but update the description of the issue here with the contents of #106, plus the content posted by @winmorgan.

**Comment by gautamjuyal** [comment]
Yes, I'm planning to work on it. I'll post here in case there is any blocker.

**Comment by mimiflynn** [comment]
I added tasks in alignment with the description from #106 which I'm including below

---

"Deeplinks" are url parameters that enable a user to share a view state.

Consistent deeplinks should be available for the following view states:

month view
week view
day view
selected event
downloadable calendar event .ics

**Comment by mimiflynn** [comment]
Thanks @gautamjuyal !

**Comment by winmorgan** [comment]
@gautamjuyal Any update on this? thanks for taking it on.

**Comment by gautamjuyal** [comment]
Sorry about the delay. I haven't made any progress on it yet. Are you planning to pick this up?

**Comment by gautamjuyal** [comment]
@winmorgan not a problem at all. I'll see if I can do anything on this issue.

**Comment by TheJuanAndOnly99** [comment]
This was completed as part of https://github.com/finos/calendar/pull/221.
